ansible_base/resource_registry/models/resource.py

```diff
@@ -40,6 +41,21 @@ def can_be_managed(self):
     def get_resource_config(self):
         return self.resource_registry.get_config_for_model(model=ContentType.objects.get_for_id(self.content_type_id).model_class())
 
+    def get_conflicting_resource(self, resource_data):
+        filter = {}
+
+        serialized_data = self.serializer_class(data=resource_data)
```

Comment:
Do you need to check `can_be_manged` to confirm there is a serializer here? Or does something else do that before calling this method?